app/views/catagories/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<div style='background-color: #3778c2; height: 100vh;'>
+
+  <div class="pt-5">
+
+    <%= form_with model: @catagory, :html => {:id => "create_catagory"} do |f| %>
```

Comment:
The user is not able to add a new category on the `categories/new` page. Moreover, please add a back button that takes the user back to the home page.

Reply:
Thank you for your recommendation. @SajjadAhmad14 
As I checked I can add a new category without any issues. Could you please tell me exactly where the issue is?